calico_node/startup/startup.go

```diff
@@ -526,7 +550,7 @@ func ipv6Supported() bool {
 
 // createIPPool creates an IP pool using the specified CIDR.  This
 // method is a no-op if the pool already exists.
-func createIPPool(client *client.Client, cidr *net.IPNet, ipipModeName string) {
+func createIPPool(client *client.Client, cidr *net.IPNet, ipipModeName string, isNATOutgoingEnabled bool) {
```

Comment:
Please update the `message` below that has `with NAT outgoing enabled` to use the passed in isNATOutgoingEnabled.  You can use %t to convert a bool into the string `true` or `false`.